draft-ietf-rats-architecture.md

```diff
@@ -759,6 +759,16 @@ could establish trust in a Verifier as discussed above.
 
 ## Attester
 
+The manufacturer of the Attester arranges for the Attester to be provisioned with key material.
+As an Endorser, this manufacturer also arranges for the Verifier to be able to verify signatures from this key material.
+
+The key material is typically in the form of asymmetric key pair (e.g., an RSA or ECDSA private key
+and a manufacturer signed IDevID certificate) secured in the Attester, with an appropriate trust anchor provided to the Verifier.
+But, these Endorsements may come in many other forms including databases of public keys (rather than certificates), or even carefully secured lists of symmetric keys.
```

Comment:
```suggestion
The key material may instead be in other forms, including databases of public keys (rather than certificates), or even carefully secured lists of symmetric keys.
```